source/standards/configuration-management.html.md.erb

```diff
@@ -7,18 +7,22 @@ expires: 2018-06-30
 
 <%= partial :expires %>
 
-Use [configuration management][] to manage, automate and standardise your infrastructure. When using configuration management you store your [infrastructure as code][], such as your virtual machines (VM) environment configuration.
+Use [configuration management][] to manage, automate and standardise your infrastructure. When using configuration management you store your [infrastructure as code][] and put that code in a version control system such as Git.
 
-A particularly good use case for configuration management is virtual machines, for example, when you run a VM in [Amazon Elastic Compute Cloud (Amazon EC2)][] or as an instance in an [Amazon Machine Image (AMI)][].This will help you track and control changes in large software deployments. You will also automatically be building environments consistently - all new environments produced from the same configuration will be identical.
+We recommend using [Puppet][] to configure servers and virtual machines. Puppet is suitable for most systems including less complex ones, is scalable and widely used within GDS. If your environment consists of a simple deployment artefact like an [Amazon Machine Image (AMI)][], Puppet may not be necessary, but you must still set out and version control the artefact's build process.
```

Comment:
How about...

 'you must still put into code and version control the artefact's build process.'

Reply:
Sure